update/Propagating_Updates.md

```diff
@@ -0,0 +1,66 @@
+# Propagating Updates
+
+A discussion on updating an existing WSO2 product deployment in a container platform.
+
+## Contents
+
+* [When do we need to update?](#when-do-we-need-to-update?)
+* [Simplest way to perform an update](#simplest-way-to-perform-an-update)
+* [Achieve zero downtime](#achieve-zero-downtime)
+
+### When do we need to update?
+
+The following are some of the scenarios which instigates the need for an update to an existing
+WSO2 product deployment in a container platform.
+
+* Integrate [WSO2 Updates](https://wso2.com/updates) containing an improvement or a bug fix
+* A version migration of the used WSO2 product
+* Updates to configuration files and non-configuration artifacts (e.g. binaries such as, third-party libraries,
+Carbon extensions in the form of OSGi bundles, [Carbon Applications](https://docs.wso2.com/display/Carbon440/Working+with+Carbon+Applications)
+or any security related artifacts such as, Java Keystore files)
+* Integrate changes to the container image source
+* Integrate changes to the container platform installation resources (e.g. Kubernetes resources or Helm Chart source)
+
+### Simplest way to perform an update
+
+We recommend the use of [official WSO2 product Helm Charts](https://hub.helm.sh/charts/wso2) for production grade WSO2 product deployments
+in Kubernetes environments.
+
+The simplest way to perform an update to an existing WSO2 product deployment is to use a [Helm based upgrade](https://helm.sh/docs/helm/helm_upgrade/).
```

Comment:
However, the simplest way to

Reply:
@chamindi-a I suppose we don't need `However` here since I am not attempting to contrast anything I have mentioned in the previous statement.

@chamindi-a @DilanUA WDYT?

Reply:
@chirangaalwis: got the impression there are two simple ways of doing updates but the simplest way was using helm updates...so if it is not contrasting points can you a conjunction to mention that it is of the same